compiler/testData/diagnostics/tests/controlFlowAnalysis/varInitializationInIfInCycle.kt

```diff
@@ -3,10 +3,10 @@ fun foo(numbers: Collection<Int>) {
     for (i in numbers) {
         val b: Boolean
         if (1 < 2) {
-            b = false
+            <!VAL_REASSIGNMENT!>b<!> = false
         }
         else {
-            b = true
+            <!VAL_REASSIGNMENT!>b<!> = true
```

Comment:
I guess this one (and below) is about the wrong CFG construction for `while` loop v.s. `for` loop where their bodies are (non) transient?

Reply:
Both of these two are wrong

Reply:
Not reported anymore.